packages/spatial/src/transform/components/TransformComponent.ts

```diff
@@ -130,7 +130,21 @@ export const TransformComponent = defineComponent({
   getMatrixRelativeToEntity: (entity: Entity, relativeEntity: Entity, outMatrix: Matrix4) => {
     const transform = getComponent(entity, TransformComponent)
     const relativeTransform = getComponent(relativeEntity, TransformComponent)
-    return outMatrix.copy(relativeTransform.matrixWorld).invert().multiply(transform.matrixWorld)
+
+    if (!transform || !relativeTransform) {
```

Comment:
because this is a hot code path, we should assume that transforms exist on both entities and handle this wherever we are unsure that they arent. otherwise a few extra ifs and a try catch could cause performance problems in the transform and physics systems.

we also want to return the outMatrix as before, such that we can use it in-line in various places

Reply:
sounds good, i'll revert this change